lib/apipie/extractor.rb

```diff
@@ -22,7 +22,7 @@ class Apipie::Railtie
         prepend Apipie::Extractor::Recorder::FunctionalTestRecording
       end
     else
-      ActionController::TestCase.send(:prepend, Apipie::Extractor::Recorder::FunctionalTestRecording)
+      ActionController::TestCase::Behavior.send(:prepend, Apipie::Extractor::Recorder::FunctionalTestRecording)
```

Comment:
The new version is not working for me. So I guess the original line was there for a reason, Why not to keep both lines in, so that it works for everyone?